base/io.jl

```diff
@@ -487,9 +487,9 @@ end
     readchomp(x)
 
 Read the entirety of `x` as a string and remove a single trailing newline.
-Equivalent to `chomp!(readstring(x))`.
+Equivalent to `chomp(readstring(x))`.
 """
-readchomp(x) = chomp!(readstring(x))
+readchomp(x) = chomp(readstring(x))
```

Comment:
Leave this out too since that's unrelated to the other change. Either we should remove `chomp!` and stop using it everywhere, or we should keep the code as-is.